zipkin-lens/src/components/App/Sidebar.js

```diff
@@ -24,6 +25,14 @@ const pageInfo = {
 };
 
 class Sidebar extends React.Component {
+  // Before Zipkin 2.13, Lens was optionally available based on a cookie named 'lens'.
+  // To revert to the classic UI, remove the cookie and reload.
+  goBackToClassic(evt) {
+    evt.preventDefault();
+    Cookies.remove('lens');
+    this.window.location.reload(true);
```

Comment:
the cookie is removed, but the page isn't reloaded...

Reply:
`this.` is not needed.

Reply:
OK. I'll fix

Reply:
lint made me put this there.. and yeah maybe that's what broke it :P